src/dmd/typesem.d

```diff
@@ -3003,6 +3003,16 @@ void resolve(Type mt, const ref Loc loc, Scope* sc, out Expression pe, out Type
             error(loc, "variable `__ctfe` cannot be read at compile time");
             return returnError();
         }
+        if (mt.ident == Id.builtin_va_list) // gcc has __builtin_va_xxxx for stdarg.h
+        {
+            /* Since we don't support __builtin_va_start, -arg, -end, we don't
+             * have to actually care what -list is. A void* will do.
+             * If we ever do care, import core.stdc.stdarg and pull
+             * the definition out of that, similarly to how std.math is handled for PowExp
+             */
+            pt = Type.tvoidptr;
```

Comment:
This line breaks gdc bootstrap, because it overrides the compilers internally defined `__builtin_va_list` type.

Example code:
```
pragma(printf)
int vfprintf(FILE* stream, scope const char* format, va_list arg);
```
This causes a compiler error because the original type of `va_list` is stomped on and replaced with `void*`.